config/webpack.config.prod.js

```diff
@@ -136,7 +136,7 @@ module.exports = {
       {
         // "file" loader makes sure those assets end up in the `build` folder.
         // When you `import` an asset, you get its filename.
-        test: /\.(jpg|png|gif|eot|svg|ttf|woff|woff2)(\?.*)?$/,
+        test: /\.(ico|jpg|png|gif|eot|svg|ttf|woff|woff2)(\?.*)?$/,
```

Comment:
Ideally `favicon.ico` specifically should end up in root build output folder so that browser see it even for files requested from the domain. (Not a big deal but still.)

We could declare a loader entry specifically for it with hardcoded `name`. However we would have to exclude `hash` from its name—otherwise it defeats the point. But then we don’t have a way to bump cache. I wonder if Webpack understands `name: 'favicon.ico?[hash:8]`.